src/amount.h

```diff
@@ -17,7 +17,7 @@ typedef int64_t CAmount;
 static const CAmount COIN = 100000000;
 static const CAmount CENT = 1000000;
 
-extern const std::string CURRENCY_UNIT;
```

Comment:
True here about the std::string duplicates; though they are all constant so cannot be changed.  I will fix this.